source/php/Component/Product/components/price.blade.php

```diff
@@ -0,0 +1,35 @@
+@foreach($prices as $price)
+    <div class="{{ $baseClass }}__price {{ $baseClass }}__price--{{ $price['backgroundColor'] ?? 'primary' }}">
+        @typography([
+            'element' => 'span',
+            'variant' => 'title',
+            'classList' => [
+                $baseClass . '__amount'
+            ]
+        ])
+            {{ $price['amount'] }}
+        @endtypography
+        @if(!empty($price['currency']))
+            @typography([
+                'element' => 'span',
+                'variant' => 'subtitle',
+                'classList' => [
+                    $baseClass . '__currency'
+                ]
+            ])
+                {{ $price['currency'] }}
+            @endtypography
+        @endif
+        @if(!empty($price['frequency']))
+            @typography([
+                'element' => 'span',
+                'variant' => 'meta',
+                'classList' => [
+                    $baseClass . '__frequency'
+                ]
+            ])
+                /{{ $price['frequency'] }}
```

Comment:
Add <span></span> around / to allow for qunique css styling of this item in the future.

Reply:
Eq if a developer wants to hide it or make it another color.